CHANGELOG.md

```diff
@@ -12,11 +12,13 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 - Create a File model which will be the base model for all the resources
   we will manage
+- create a property `RESOURCE_NAME` on models having a url
 
 ## Changed
 
 - Refactor the LTI view to be generic for all the resources we want to manage
 - Video model is a special File model
+- pluralize thumbnail url. This a BC break
```

Comment:
not really BC break if you consider the API to be internal to the product... it is not a public API.
This being said, we are moving to v3 so whatever...